src/components/Dialogs/ImageInfo/ImageInfoDialogComponent.scss

```diff
@@ -0,0 +1,38 @@
+.image-info-dialog {
+    min-width: 60%;
+    height: 100%;
+    user-select: none;
+
+    .bp3-dialog-body {
+        height: 100%;
+        overflow: hidden;
+        margin-top: 10px;
+        margin-left: 25px;
```

Comment:
don't think we need such a large left margin. In any case, it should be symmetric

Reply:
This is a legacy from FileBrowserComponent. I will adjust it.